pydfs_lineup_optimizer/rules.py

```diff
@@ -364,6 +364,7 @@ def apply(self, solver, players_dict):
         for_positions = self.optimizer.team_stacks_for_positions
         if for_positions:
             all_players = [player for player in all_players if list_intersection(player.positions, for_positions)]
+            all_players.strip('P')
```

Comment:
Why do you need this? Does it works? because `all_players` is a normal python list and it hasn't strip function.

Reply:
This removes the Pitcher from the pool of players chosen for stacks. Theoretically you don't want to stack a pitcher with batters because they have no correlation when it comes to fantasy points. You want to stack batter with one another because of their positive correlation.

Reply:
I think what you are looking for already implemented in version 2.6.0 In this version I've added additional parameter `for_positions` to stacking method, so you can specify positions used in stack like `optimizer.set_team_stacking([3, 3], for_positions=['1B', '2B', '3B', 'C', 'SS', 'OF'])`

Reply:
ah ok that works

Reply:
In this case I'll close your PR because I think more users use optimizer for DK and export for DK is more preferable, for FD you can write your custom function and pass it to export function as I wrote in comment above.